app/src/main/res/layout/view_sliding_header.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Useless LinearLayout, remove it

Reply:
I need it in future to place there detailed info about song